rollup.config.js

```diff
@@ -162,11 +174,6 @@ if (process.env.SERVER === 'true') {
       port: process.env.PORT || 9100
     })
   )
-  plugins.push(
-    livereload({
-      watch: 'dist'
-    })
-  )
```

Comment:
Why did you remove livereload?

If you remove it here for a reason, then the imports and references in package.json and yarn.lock should be removed as well.

Reply:
Hey, thanks for the hint and yes technically you're right. This is/was a draft PR and something about livereload broke the build process (and we didn't bother investigating there since it was a hacky "let's see how far we get" anyways)

Will get fixed in the cleanup if/once we continue with this PR ;)